openfecli/commands/plan_rbfe_network.py

```diff
@@ -126,7 +126,7 @@ def plan_rbfe_network(
     write("")
 
     write("Using Options:")
-    mapper_obj = MAPPER.get(mapper)()
+    mapper_obj = LomapAtomMapper(element_change=False)
```

Comment:
```suggestion
    mapper_obj = LomapAtomMapper(time=20, threed=True, element_change=False, max3d=1)
```
This is what we use in openfe-benchmarks by default.